Documentation/DoxPages/mainpage.dox

```diff
@@ -1,5 +1,6 @@
 /*!
  * \mainpage
  *
- * Go to the <A HREF="../index.html">main Katydid documentation</A>.
-*/
\ No newline at end of file
+ * Go to the <A HREF="../index.html">main Katydid documentation</A> for documentation of specific classes and functions.
```

Comment:
I don't think this change makes sense since it's the API documentation in doxygen (where this link shows up) that has information about specific classes and functions, not the main doc site that this links to.

Reply:
Sure, we can take that out  (it was there when we started, we didn't add it so I'm guessing we're none of us too attached to it)

Reply:
remved in f8686f96ad7f6720724b8b9dac834296e922d775

Reply:
right, the actual way to write that commit link: f8686f9